cmd/agent/app/agent.go

```diff
@@ -27,20 +27,20 @@ import (
 
 	"go.uber.org/zap"
 
-	"github.com/uber/jaeger/cmd/agent/app/processors"
+	bProcessors "github.com/uber/jaeger/cmd/agent/app/processors"
```

Comment:
this change looks unnecessary. If it clashes with the new `processors` variable then we can rename the var to `defaultProcessors`, it's better than renaming the package